recipe/meta.yaml

```diff
@@ -12,6 +12,8 @@ source:
 
 build:
   number: 2
```

Comment:
Build number should be `0` if we build a new version of the package

Reply:
The build number has been updated:
https://github.com/AnacondaRecipes/tifffile-feedstock/blob/update_to_2021.11.2/recipe/meta.yaml#L14